samtranslator/swagger/swagger.py

```diff
@@ -408,14 +409,18 @@ def set_path_default_authorizer(self, path, default_authorizer, authorizers):
         :param string default_authorizer: Name of the authorizer to use as the default. Must be a key in the
             authorizers param.
         :param list authorizers: List of Authorizer configurations defined on the related Api.
+        :param bool add_default_auth_to_preflight: Bool of whether to add the default
+            authorizer to OPTIONS preflight requests.
         """
 
         for method_name, method in self.get_path(path).items():
+            normalized_method_name = self._normalize_method_name(method_name)
             # Excluding paramters section
-            if method_name == "parameters":
+            if normalized_method_name == "parameters":
                 continue
-            self.set_method_authorizer(path, method_name, default_authorizer, authorizers,
-                                       default_authorizer=default_authorizer, is_default=True)
+            if not (add_default_auth_to_preflight is False and normalized_method_name == "options"):
```

Comment:
This is a little confusing. Isn't the following functionally equivalent?

`if add_default_auth_to_preflight or normalized_method_name != "options"`:

Reply:
Can we make this change